pkg/controller/pravegacluster/pravegacluster_controller.go

```diff
@@ -151,6 +158,7 @@ func (r *ReconcilePravegaCluster) run(p *pravegav1alpha1.PravegaCluster) (err er
 }
 
 func (r *ReconcilePravegaCluster) deployCluster(p *pravegav1alpha1.PravegaCluster) (err error) {
+
```

Comment:
These new lines after the function signature don't seem to be added consistently.

Reply:
removed